package-lock.json

```diff
@@ -22034,7 +22035,6 @@
       "version": "1.18.1",
       "resolved": "https://registry.npmjs.org/http-proxy/-/http-proxy-1.18.1.tgz",
       "integrity": "sha512-7mz/721AbnJwIVbnaSv1Cz3Am0ZLT/UBwkC92VlxhXv/k/BBQfM2fXElQNC27BVGr0uwUpplYPQM9LnaBMR5NQ==",
-      "dev": true,
```

Comment:
I'm not sure why this is no longer "dev": true, as this wasn't changed. Please let me know if I should omit this change (likewise below)